standards-catalogue/Rakefile

```diff
@@ -51,6 +60,24 @@ namespace :schema do
     end
     raise unless errors.all?
   end
+
+  desc "Validate organisations are schema compliant"
+  task :organisations do
+    schema = {
+      "type" => "object",
+      "required" => %w[name url],
+      "properties" => {
+        "name" => { "type" => "string" },
```

Comment:
we should make sure this has a minimum lenght - right now an empty string could be used